libdevcore/CommonData.h

```diff
@@ -334,4 +334,11 @@ bool containerEqual(Container const& _lhs, Container const& _rhs, Compare&& _com
 	return std::equal(std::begin(_lhs), std::end(_lhs), std::begin(_rhs), std::end(_rhs), std::forward<Compare>(_compare));
 }
 
+std::string findAnyOf(std::string const& _haystack, std::vector<std::string> const& _needles)
```

Comment:
I think this must be marked `inline` because it is not a template.

Reply:
done.